### PR TITLE
Rebuild 3.6

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
@conda-forge-admin please rerender

**Comment by mariusvniekerk** [comment]
@jjhelmus Think this is probably sane mostly.  Copied over the mechanics from the 3.7 recipe

**Comment by mariusvniekerk** [comment]
This seems to be around build string matching

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,7 +9,8 @@ source:
   - url: https://www.python.org/ftp/python/{{ version }}/Python-{{ version }}.tgz
     sha256: {{ sha256 }}
     patches:
-      - 0001-Add-Anaconda-Distribution-version-logic.patch
+      # This does a weird thing on conda-forge.  disable for now.
```
Comment (ocefpaf) [inline recipe/meta.yaml]: It would be nice to write here what is the weird thing :smile:

**Pull request merged**